UnityProject/Assets/Scripts/Core/Chat/ChatUI.cs

```diff
@@ -607,6 +625,7 @@ public void OnMouseExit()
 		{
 			if (UIManager.IsInputFocus) return;
 			Showing = false;
+			STTImageText.color = Color.white;
```

Comment:
All of this seems like it should be managed by its own component, and not be a part of the chat UI, since it is not its responsibility.

Reply:
well the chat you're has its own button to toggle the SST since you can't press m

Reply:
and you can't have a component that references the buton because?

Reply:
Was quicker to do through the chatUI

Reply:
also resetting when the caht UI closes

Reply:
what?

Reply:
this button is for the chat you only it only shows when you open the cchat